src/NHibernate.Test/CfgTest/ConfigurationSchemaFixture.cs

```diff
@@ -35,6 +35,23 @@ public void FromAppConfigTest()
 			Assert.IsTrue(hc.UseReflectionOptimizer);
 			Assert.AreEqual("NHibernate.Test", hc.SessionFactory.Name);
 		}
+ 
+		[Test]
+		public void ByteCodeProvider()
```

Comment:
The test one above will fail.

Reply:
I don't understand...

Reply:
The first test of the fixture still succeed because the configuration is invalid by lack of `session-factory` node. That is not a schema validation error, but a hard-coded check in the parsing.

Reply:
I understood that later, but did not report it here. Can you rename that test to something more meaningful than `InvalidConfig`?